src/OpenSage.Game/Client/Drawable.cs

```diff
@@ -20,6 +20,8 @@ private void AddDrawModule(string tag, DrawModule module)
             module.Tag = tag;
             _drawModules.Add(module);
             _tagToModuleLookup.Add(tag, module);
+
+            AddDisposable(module);
```

Comment:
The only concern I have is this is pretty core to like... The whole game. Maybe we've been missing this behavior the whole time though?

For my own sanity I'd like to do some cursory playtests to make sure we're not breaking something weird here (or maybe this will solve a bunch of problems instead, who knows!)